paywall/src/data-iframe/cache/localStorage.js

```diff
@@ -1,7 +1,9 @@
 import localStorageAvailable from '../../utils/localStorage'
 
-export function storageId(networkId, accountAddress) {
-  return `unlock-protocol/${networkId}/${accountAddress}`
+export function storageId(networkId, accountAddress, type) {
```

Comment:
this is the primary change: modifying the storage id to append a type if specified